tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/AnimalProduct.java

```diff
@@ -0,0 +1,102 @@
+package com.sci.bv.tema14Ianuarie;
+
+
+/**
+ * In this Class are stocked animal products and their characteristics.
+ * (ID, quantity,name of product, list of vitamins, price, validity, weight)
+ */
+
+public class AnimalProduct {
+
+    private int ID;
+    private int Qty;
+    private String Name;
+    private double Temp;
+    private double Price;
+    private int Validity;
+    private double Weight;
+
+    public AnimalProduct() {
+        ID = 12345;
+        Qty = 45;
+        Name = "Branza de vaci";
+        Temp = 8.5;
+        Price = 5;
+        Validity = 20190305;
+        Weight = 1;
```

Comment:
This might be useful for testing, but it's not ok to have test data deep in your code. you can have a method "createDummyAnimalProduct()" which can create an AnimalProduct with these values if you want. That method should be next to your testing code (next to your main method).